plugin/src/main/scala/sbtnativeimage/NativeImagePlugin.scala

```diff
@@ -155,6 +190,25 @@ object NativeImagePlugin extends AutoPlugin {
         }
       }
     }.value,
+    nativeImageAgentOutputDir := baseDirectory.value / "native-image-configs",
```

Comment:
Are users expected to commit the output from the native-image agent? By convention, `baseDirectory` is normally tracked by git
```suggestion
    nativeImageAgentOutputDir := targetDirectory.value / "native-image-configs",
```